Kalela/BrightEventsReactApp
# Ft-react app-155330585

### What does this PR do?
Sets up basic components for the bright events react frontend.
### How should this be manually tested?
- Clone the repo locally.
- cd into the project.
- Make sure you have npm installed. Check installation [here](https://www.codecademy.com/articles/react-setup-i)
- Run ```npm install -g``` to install all the packages documented in package.json.
- Run ```npm start``` in your terminal to get the server up.
- Visit the site on ```localhost:3000``` and try out the basic functionality.
### What are the relevant pivotal tracker stories?
Set up react app
ID: #155330585
